File: tests/fixtures/sample/src/sample.py
def helper(value: int) -> int:
    """Return value plus one."""
    return value + 1


class Greeter:
    """Greeter docstring."""

    def __init__(self, name):
        self.name = name

    @staticmethod
    def message(name: str) -> str:
        """Compose greeting."""
        return f"hello {name}"


def add(a, b=1):
    return a + b

File: tests/fixtures/sample/src/sample.swift
import Foundation

/// Greeter for Swift samples.
class Greeter {
  /// Compose greeting.
  @available(iOS 13.0, *)
  func sayHello(name: String) -> String {
    return "hello \(name)"
  }
}

struct Counter {
  var value: Int

  init(start: Int) {
    self.value = start
  }
}
